networking/ingress/configuring-ingress-operator.adoc

```diff
@@ -5,15 +5,13 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-The Ingress Operator is an OpenShift component that enables external access to
-cluster services by configuring Ingress Controllers. You can use the Ingress
-Operator to route traffic by specifying OpenShift `Route` and Kubernetes `Ingress`
-resources.
-
-The Ingress Operator deploys and manages an OpenShift router, which is a
-HAProxy-based Kubernetes Ingress controller.
-
-Ingress Operator implements the OpenShift `ingresscontroller` API.
+The Ingress Operator implements the `ingresscontroller` API and is the
+component responsible for enabling external access to OpenShift
+cluster services. The Operator makes this possible by deploying and
```

Comment:
should Operator be capitalized here?

Reply:
Thanks for catching this. I changed to lowercase and updated the PR.